benchmark/opperf/nd_operations/binary_operators.py

```diff
@@ -38,7 +38,7 @@
     get_all_elemen_wise_binary_operators
 
 
-def run_mx_binary_broadcast_operators_benchmarks(ctx=mx.cpu(), dtype='float32', warmup=10, runs=50):
+def run_mx_binary_broadcast_operators_benchmarks(ctx=mx.cpu(), dtype='float32', warmup=25, runs=100):
```

Comment:
Is this default warmup/runs applicable to all operators? If yes, maybe we can use a global constant so that it becomes consistent for all operators?

Reply:
For now yes, but, I see it to be different. For example - For Convolution operator variance is close to 0 within 25 warmup and 50 runs. For simple op like log/tan etc., we need to run 100+ runs to reduce variance. So made it configurable per operator category